app/upgrades/v2/upgrades.go

```diff
@@ -0,0 +1,86 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"cosmossdk.io/log"
+	sdkmath "cosmossdk.io/math"
+	storetypes "cosmossdk.io/store/types"
+	circuittypes "cosmossdk.io/x/circuit/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	"github.com/andromedaprotocol/andromedad/app/upgrades"
+)
+
+const UpgradeName = "v2"
+
+// NewUpgrade constructor
+func NewUpgradeV2() upgrades.Upgrade {
+	return upgrades.Upgrade{
+		UpgradeName:          UpgradeName,
+		CreateUpgradeHandler: CreateV2UpgradeHandler,
+		StoreUpgrades: storetypes.StoreUpgrades{
```

Comment:
Still need clarification if Alliance is meant to be removed given Luna/TFL situation

Reply:
Alliance module can be removed